src/test/e2e/50_oci_publish_deploy_test.go

```diff
@@ -48,7 +48,6 @@ func (suite *PublishDeploySuiteTestSuite) Test_0_Publish() {
 	ref := suite.Reference.String()
 	stdOut, stdErr, err := e2e.Zarf(suite.T(), "package", "publish", example, "oci://"+ref, "--plain-http")
 	suite.NoError(err, stdOut, stdErr)
-	suite.Contains(stdErr, "Published "+ref)
```

Comment:
Checking stderr for a specific string feels like a change detector -- any actual error in the process should be caught by the suite.NoError above. Initially I set these to match in both message and logger, but upon more consideration we should just delete these checks and make sure our error trees work. Thoughts?